packages/fxa-payments-server/src/App.tsx

```diff
@@ -48,7 +48,7 @@ export const App = ({
   };
   return (
     <StripeProvider apiKey={config.stripe.apiKey}>
-      <Provider store={store}>
+      <ReduxProvider store={store}>
```

Comment:
Using more React contexts around here, so I figured renaming this to `ReduxProvider` would help readability.